opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc

```diff
@@ -6,7 +6,8 @@
 === Configuration Tester
 
 To identify configuration problems there is a config-tester located in `$OPENNMS_HOME/bin/`.
-The `config-tester` runs while OpenNMS starts and checks a number of configuration files. Type `-l, --list` to view the list of files checked. 
+The `config-tester` runs while OpenNMS starts and checks a number of configuration files. 
```

Comment:
The `config-tester` does not run automatically when OpenNMS starts. The user has to run it by himself to test his configuration files.

Reply:
Updated.